assets/scripts/submit.js

```diff
@@ -36,7 +36,8 @@
     if (!errorPanel) {
       errorPanel = document.createElement('div')
       errorPanel.classList.add('errors')
-      loginForm.appendChild(errorPanel)
+      // loginForm.prepend(errorPanel)
```

Comment:
Can you remove this commented line?

Reply:
Sorry, I removed it in the end and forgot to save before commiting and pushing… (¬_¬)